packages/box/test/box.js

```diff
@@ -67,7 +69,7 @@ describe("@truffle/box Box", () => {
       });
     });
 
-    it("unboxes truffle box from local folder", async () => {
+    it("unboxes truffle box from local folder [win]", async () => {
```

Comment:
This tag doesn't actually do anything, right?  Is that why you didn't include the `@` sign?  I'm thinking it should probably be included for consistency even though it doesn't actually do anything atm, and you can just leave a comment about how it does nothing?  Or something?

Reply:
It doesn't belong here. Removed!

Reply:
OK, if you say so!